docs/tutorial/part-eight/index.md

```diff
@@ -1,94 +1,100 @@
 ---
-title: Preparing a Site to Go Live
+title: Preparando um site para ir ao ar
 typora-copy-images-to: ./
 disableTableOfContents: true
 ---
 
-Wow! You've come a long way! You've learned how to:
+Uau! você chegou bem longe! Você aprendeu até o momento:
 
-- create new Gatsby sites
-- create pages and components
-- style components
-- add plugins to a site
-- source & transform data
-- use GraphQL to query data for pages
-- programmatically create pages from your data
+- criar novos sites com Gatsby
+- criar páginas e componentes
+- estilizar componentes
+- adicionar plugins ao site
+- código & tratando dados
+- usar GraphQL para consultar dados para a página
+- criar páginas automaticamente a partir dos seus dados
 
-In this final section, you're going to walk through some common steps for preparing a site to go live by introducing a powerful site diagnostic tool called [Lighthouse](https://developers.google.com/web/tools/lighthouse/). Along the way, we'll introduce a few more plugins you'll often want to use in your Gatsby sites.
+Nessa seção final, você passará por alguns dos passos comuns ao preparar um site ao ir para ir ao ar, introduzindo uma poderosa ferramenta chamada [Lighthouse](https:/developers.google.com/web/tools/lighthouse/). Ao longo do caminho nós iremos introduzir alguns plugins a mais que você possa vir a usar com frequência nos seus sites com Gatsby.
```

Comment:
"https:/" => "https://"

Reply:
All the links in the original article are using `https://` so it's not related to the translation. Should I consider it?